sdk/appcomplianceautomation/Azure.ResourceManager.AppComplianceAutomation/src/Generated/Models/AppComplianceReportPatchProperties.cs

```diff
@@ -0,0 +1,133 @@
+        /// <param name="subscriptions"> List of subscription Ids. </param>
+        /// <param name="complianceStatus"> Report compliance status. </param>
+        /// <param name="storageInfo"> The information of 'bring your own storage' binding to the report. </param>
+        /// <param name="certRecords"> List of synchronized certification records. </param>
+        /// <param name="provisioningState"> Azure lifecycle management. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal AppComplianceReportPatchProperties(DateTimeOffset? triggerOn, string timeZone, IList<ReportResourceMetadata> resources, AppComplianceReportStatus? status, IReadOnlyList<string> errors, Guid? tenantId, string offerGuid, DateTimeOffset? nextTriggerOn, DateTimeOffset? lastTriggerOn, IReadOnlyList<string> subscriptions, ReportComplianceStatus complianceStatus, ReportStorageInfo storageInfo, IReadOnlyList<CertSyncRecord> certRecords, AppComplianceProvisioningState? provisioningState, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            TriggerOn = triggerOn;
+            TimeZone = timeZone;
+            Resources = resources;
+            Status = status;
+            Errors = errors;
+            TenantId = tenantId;
+            OfferGuid = offerGuid;
+            NextTriggerOn = nextTriggerOn;
+            LastTriggerOn = lastTriggerOn;
+            Subscriptions = subscriptions;
+            ComplianceStatus = complianceStatus;
+            StorageInfo = storageInfo;
+            CertRecords = certRecords;
+            ProvisioningState = provisioningState;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Report collection trigger time. </summary>
+        public DateTimeOffset? TriggerOn { get; set; }
+        /// <summary>
+        /// Report collection trigger time's time zone, the available list can be obtained by executing "Get-TimeZone -ListAvailable" in PowerShell.
+        /// An example of valid timezone id is "Pacific Standard Time".
+        /// </summary>
+        public string TimeZone { get; set; }
+        /// <summary> List of resource data. </summary>
+        public IList<ReportResourceMetadata> Resources { get; }
+        /// <summary> Report status. </summary>
+        public AppComplianceReportStatus? Status { get; }
+        /// <summary> List of report error codes. </summary>
+        public IReadOnlyList<string> Errors { get; }
+        /// <summary> Report's tenant id. </summary>
+        public Guid? TenantId { get; }
+        /// <summary> A list of comma-separated offerGuids indicates a series of offerGuids that map to the report. For example, "00000000-0000-0000-0000-000000000001,00000000-0000-0000-0000-000000000002" and "00000000-0000-0000-0000-000000000003". </summary>
+        public string OfferGuid { get; set; }
```

Comment:
Looks like this should be of type `Guid`?

Reply:
Not all string, so keep it as string